commit 73614e206a3ee822ef2ad1ac648a4616600ab87e (refs/changes/65/3865/3)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-05-01T19:02:50+00:00 (1 year, 5 months ago)
    
    OS-6827 Update vmadm to capture new log after OS-6718
    Reviewed by: Josh Wilsdon <josh.wilsdon@joyent.com>
    Approved by: Josh Wilsdon <josh.wilsdon@joyent.com>

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index c8c74997..d82ab6d8 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -6564,6 +6564,16 @@ function archiveVM(uuid, options, callback)
                 dst: archive_dirname,
                 targ: path.join(archive_dirname, 'cores')
             });
+            patterns_to_archive.push({
+                src: path.join(vmobj.zonepath, 'logs/console.log*'),
+                dst: path.join(archive_dirname, 'console'),
+                create_dst_dir: true
+            });
+            patterns_to_archive.push({
+                src: path.join(vmobj.zonepath, 'logs/platform.log*'),
+                dst: path.join(archive_dirname, 'platform'),
+                create_dst_dir: true
+            });
 
             if (vmobj.brand === 'kvm') {
                 patterns_to_archive.push({
@@ -6576,17 +6586,6 @@ function archiveVM(uuid, options, callback)
                     dst: archive_dirname,
                     targ: path.join(archive_dirname, 'startvm')
                 });
-            } else if (vmobj.brand === 'bhyve') {
-                patterns_to_archive.push({
-                    src: path.join(vmobj.zonepath, 'root/tmp/zhyve.log'),
-                    dst: path.join(archive_dirname, 'vmlogs'),
-                    create_dst_dir: true
-                });
-                patterns_to_archive.push({
-                    src: path.join(vmobj.zonepath, 'root/var/run/*.log*'),
-                    dst: path.join(archive_dirname, 'vmlogs'),
-                    create_dst_dir: true
-                });
             } else if (vmobj.docker) {
 
                 function logTimestamp() {
